README.md

```diff
@@ -66,10 +66,37 @@ Nx comes with local caching already built-in (check your `nx.json`). On CI you m
 Before any publish, releasing is mandatory if you want it to take any effect.
 
 1. Release
+
+You will need to be connected to github (GITHUB_TOKEN, GH_TOKEN) or with cli login.
+```
+sudo apt install gh
```

Comment:
Ajouter aussi avec homebrew pour mac (pas réussi avec apt)

install homebrew if you don't already have it
```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
install gh
```
brew install gh
```

Reply:
you can update it if you see a problem with a documentation / comment.